README.md

```diff
@@ -64,7 +64,7 @@ If you already got the URL of the exact resource you want, you can download it d
 ```console
 $ annie https://img9.bcyimg.com/drawer/15294/post/1799t/1f5a87801a0711e898b12b640777720f.jpg
 
-annie doesn't support this URL by now, but it will try to download it directly
+annie doesn't support this URL right now, but it will try to download it directly
```

Comment:
sorry for this, it can't be changed here, the output comes from Annie inside

Reply:
Ahh, sorry about that.